### PR TITLE
Implement individual dice roll display for Roll for Shoes

### PR DESCRIPTION
**Pull request by Carl-Rocha**
# Submission Checklist
## Required

- [x] The pull request title clearly contains the name of the sheet I am editing.
- [X] The pull request title clearly states the type of change I am submitting (New Sheet/New Feature/Bugfix/etc.).
- [X] The pull request makes changes to files in only one sub-folder.
- [X] The pull request does not contain changes to any json files in the translations folder (translation.json is permitted)

## New Sheet Details

- The name of this game is: <   >  _(i.e. Dungeons & Dragons 5th Edition, The Dresden Files RPG)_
- The publisher of this game is: <   > _(i.e. Wizards of the Coast, Evil Hat)_
- The name of this game system/family is: <   > _(i.e. Dungeons & Dragons, FATE)_

- [x] I have followed the [Character Sheets Standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) when building this sheet.

- [ ] I have authorization from the game's publisher to make this an official sheet on Roll20 with their name attached.
- [ ] This game is not a traditionally published game, but a copy of the game rules can be purchased/downloaded/found at: <   >
- [ ] This sheet is for an unofficial fan game, modification to an existing game, or a homebrew system.

# Changes / Description

This pull request introduces a new feature to the "Roll for Shoes" character sheet that allows users to add a modifier to their skill checks. The key changes include:

Skill Modifier Input: Added a prompt for users to input a modifier value, which will be applied to each dice roll in skill checks.
Individual Dice Roll Display: Modified the skill check logic to display each dice roll individually in the roll template, providing clearer feedback to users.
Bug Fix: Resolved an issue where skill checks with a negative rating were not calculating correctly.
These changes enhance the functionality of the "Roll for Shoes" sheet, making it more versatile and user-friendly. The implementation follows the existing coding standards for Roll20 character sheets.

### CODE REVIEW

**Comment by NBrooks-Roll20** [comment]
hi @Carl-Rocha! As this sheet was just created a few months ago by @joshforisha, we're going to give them a chance to review this and approve or deny this update to their sheet.

File: Roll for Shoes/sheet.html [hunk 1]
```diff
@@ -26,46 +26,46 @@ <h3>Statuses</h3>
...
-        ? `${rating}d6`
-        : `${Math.abs(rating)}d6kl1`
+          let rollTemplate = `&{template:default} {{name=${skillName}}}`
 
-      startRoll(`&{template:default} {{name=${skillName} ${rating}}} {{result=[[${rollQuery}+?{Modifier}]]}}`, ({ rollId }) => {
-        finishRoll(rollId)
+          const modifier = `+?{Modifier|0}`;
+
+          for (let i = 1; i <= Math.abs(rating); i++) {
+              rollTemplate += ` {{roll${i}=[[1d6${modifier}]]}}`
```
Comment (joshforisha) [inline Roll for Shoes/sheet.html:51]: Doesn't this add a modifier for *each* die rolled? That's not how statuses and modifiers should work in the system.

Comment (joshforisha) [reply Roll for Shoes/sheet.html:51]: This also removes the possibility of negative skills, which are an option (as Flaws) and should remain possible.

**Review by joshforisha — CHANGES_REQUESTED** [review]
Some issues with the skill rolling, and indentation is all wonky now.

**Comment by joshforisha** [comment]
@Carl-Rocha I've taken a pass at implementing this feature in #13251

**Comment by kfroll20** [comment]
@Carl-Rocha, @joshforisha has implemented the requested changes in their PR so we are going to close this one. 
Thank you for your contribution!

**Pull request closed without merging**